remoting/subsystem/src/main/java/org/jboss/as/remoting/HttpConnectorAdd.java

```diff
@@ -72,4 +81,18 @@ void launchServices(OperationContext context, String connectorName, ModelNode mo
         final String saslAuthenticationFactory = saslAuthenticationFactoryModel.isDefined() ? saslAuthenticationFactoryModel.asString() : null;
         RemotingHttpUpgradeService.installServices(context, connectorName, connectorRef, RemotingServices.SUBSYSTEM_ENDPOINT, optionMap, securityRealm, saslAuthenticationFactory);
     }
+
+    void validateDuplicatedResources(OperationContext context, PathAddress address, String connectorName, Resource resource) throws OperationFailedException {
+        PathAddress parentAddress = address.getParent();
+        Resource parent = context.readResourceFromRoot(parentAddress, false);
+        ModelNode resourceRef = resource.getModel().get(CommonAttributes.CONNECTOR_REF);
+        boolean sameUndertowListener = parent.getChildren(CommonAttributes.HTTP_CONNECTOR).stream()
```

Comment:
Please just use a for loop. 3 lambdas = over 2K in code space vs a couple of if statements.